tuning/autotune.py

```diff
@@ -172,6 +176,23 @@ def run_command(
             raise
 
 
+def parse_xtune(Xtune: list[str]) -> dict[str, str]:
+    """Parse Xtune array to dict
+    e.g. ['--lhs-dims=bmk', '--rhs-dims=bkn', '--tile-dims=*mnk'] -> {'lhs-dims': 'bmk', 'rhs-dims': 'bkn', 'tile-dims': '*mnk'}
+    """
+    args_dict = {}
+
+    for arg in Xtune:
+        parts = arg.split("=")
+        if len(parts) == 2:
+            key, value = parts
+            args_dict[key.strip("-")] = value
+        elif len(parts) == 1:
+            args_dict[parts[0].strip("-")] = None
+
+    return args_dict
```

Comment:
At this point we can make these regular flags to autotune.sh -- we know exactly what we need because we call the tune function directly